src/hatchet/utils/count_alleles.py

```diff
@@ -191,7 +191,7 @@ def counting(
     )
 
     results = worker.run(work=work, n_instances=num_workers)
-    results = {(v[0][0], v[0][1]): v for v in results}
+    results = {(v[0][0], v[0][1]): v for v in results if v}
```

Comment:
This oversight on my part (introduced in v1.0.2) was the cause of issue #150. This is the bug fix.